### DISCUSSION TITLE
제페토 앱에서 스크립트가 비활성화됨

### DISCUSSION THREAD

**Discussion by noil-zepe** [naverz/zepeto-studio-kor#953] [5. Q&A]
안녕하세요!  제페토에서 월드를 만들기 위해 공부하고 있습니다.

WaterProDaytime을 사용하여 수면을 만들었습니다.
  unity 내에서는 잘 반영되고 있습니다만, ZEPETO 앱으로 프리뷰를 실시하면, water의 스크립트가 무효화되어 버립니다.

WaterProDaytime이 TypeScript가 아니기 때문입니까?

나는 프로그래밍도 처음이며 완전히 초보자입니다.  게다가 번역기를 사용하고 있습니다.  무례하면 죄송합니다.  잘 부탁드립니다.
![6137D1D8-9B90-435B-AD66-547E4368FEA3](https://user-images.githubusercontent.com/102274326/177096634-4520e739-9088-4356-aadb-ccda1185ed91.png)
![B06C464E-022B-4A3F-8B91-08D26EA0941A](https://user-images.githubusercontent.com/102274326/177096654-d8a99ee6-7eda-470c-90d3-87226d867127.jpeg)

**Comment by YSHFM** [comment]
Zepeto can't used c#script. you can only used typescript.

**Reply by noil-zepe** [reply]
대답 해 주셔서 감사합니다. 기뻐요. 먼저 자바 스크립트를 배우고 그 다음에 TypeScript를 배웁니다 (TT)

TypeScript에서도 반사를 만들 수 있습니까?

**Reply by YSHFM** [reply]
Yes, with shader and material.

**Reply by noil-zepe** [reply]
thank you for answering!  Examine the reflective shader and material.  

Finally, let me ask another question..
  I don't think most assets in the asset store have TypeScript, but if you use assets, do you convert them to TypeScript and use them?  Or is it basically impossible to use assets from the Asset Store?

**Reply by YSHFM** [reply]
you can use assets from the asset store. 
If the C# script is mixed in assets, you need to convert it and use it.

**Reply by noil-zepe** [reply]
감사합니다.  변환할 수 있도록 공부!  여러분 덕분에 저는 월드 제작을 계속하겠습니다.
